5-infrastructure/modules/base-env/providers.tf

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+provider "google" {
```

Comment:
We should use native impersonation via https://github.com/terraform-google-modules/terraform-example-foundation/issues/281

Reply:
@bharathkkb can you link a provider file maybe using native impersonation as an example?

Reply:
https://github.com/terraform-google-modules/terraform-example-foundation/blob/1306710f24549ae84547f5c040ee058b6280a287/5-app-infra/business_unit_1/development/providers.tf as an example